templates/taxi/car_detail.html

```diff
@@ -3,20 +3,37 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    {% if user not in car.drivers.all %}
```

Comment:
The condition `{% if user not in car.drivers.all %}` may not work as expected because `user` is a Django `User` object, and `car.drivers.all` returns a queryset of `Driver` objects. Ensure that `user` is correctly compared with the `Driver` objects, possibly by checking a specific attribute like `user.id` against `driver.user_id`.

Reply:
its working correct, thanks